integrationTests/vm/arwen/testdata/bad-misc/elrond/bigInt.h

```diff
@@ -20,9 +23,14 @@ long long bigIntGetInt64(bigInt reference);
 void      bigIntSetInt64(bigInt destination, long long value);
 
 void      bigIntFinishUnsigned(bigInt reference);
+void      bigIntFinishSigned(bigInt reference);
 void      bigIntGetCallValue(bigInt destination);
-void      bigIntGetExternalBalance(byte *address, bigInt result);
+void      bigIntgetExternalBalance(byte *address, bigInt result);
```

Comment:
Get instead get ?

Reply:
I checked in Arwen and it is indeed with a capital G

Reply:
Fixed.